os/Makefile.unix

```diff
@@ -464,6 +464,10 @@ ifeq ($(CONFIG_SUPPORT_COMMON_BINARY),y)
 		mkdir -p "$(TOPDIR)/../tools/fs/contents-smartfs/$(CONFIG_ARCH_BOARD)"; \
 		mkdir -p "$(TOPDIR)/../tools/fs/contents-smartfs/$(CONFIG_ARCH_BOARD)/base-files"; \
 	fi
+ifeq ($(CONFIG_COMPRESSED_BINARY),y)
+	$(Q) tools/compression/mkcompressimg $(CONFIG_COMPRESSION_BLOCK_SIZE) $(CONFIG_COMPRESSION_TYPE) $(OUTBIN_DIR)/$(CONFIG_COMMON_BINARY_NAME) $(OUTBIN_DIR)/$(CONFIG_COMMON_BINARY_NAME).comp
```

Comment:
It makes compressed binary with `.comp` and change the name to the origin.
Is it possible to make compressed binary with `CONFIG_COMMON_BINARY_NAME` directly?

Reply:
No.. Its not possible. The compression utility requires output file to be different from input file.

Reply:
Okay. Fine. Thanks.